contrib/IECoreArnold/src/IECoreArnold/UniverseBlock.cpp

```diff
@@ -53,7 +53,12 @@ namespace
 void loadMetadata( const std::string &pluginPaths )
 {
 	typedef boost::tokenizer<boost::char_separator<char> > Tokenizer;
-	Tokenizer t( pluginPaths, boost::char_separator<char>( ":" ) );
```

Comment:
I'm surprised this is the only spot we've hardcoded `:` separator, have you looked for others?

Reply:
> If you're looking for these changes to be Gaffer 0.59 compatible I'd recommend targetting `RB-10.1` rather than `master`.

I'm happy with it going into 10.2 and Gaffer 0.60 - you're right that I'm using a parallel track for my Gaffer builds that I'm cherry-picking from to create organized PRs here.

> I'm surprised this is the only spot we've hardcoded `:` separator, have you looked for others?

Good catch - there are a few others that I added in 27f541988e5932d6ea9b8630b95cf90c7dd81110. I didn't change the SearchPath test yet, it seems like it's still testing the appropriate Linux functionality and I can take care of the Windows side as part of my test passing efforts.